src/heading.js

```diff
@@ -111,4 +111,6 @@ export default class Heading extends Plugin {
  * @property {module:engine/view/elementdefinition~ElementDefinition} view Definition of a view element to convert from/to.
  * @property {String} title The user-readable title of the option.
  * @property {String} class The class which will be added to the dropdown item representing this option.
+ * @property {String} [icon] Icon used by {@link module:heading/headingbuttonsui~HeadingButtonsUI}. It can be omitted when using
+ * the default configuration.
```

Comment:
Do we need this option? I mean: there is no such option for basic styles, why there is such option for headings?

Reply:
It is needed in my opinion because heading plugin can be configurable - so if someone provides configuration for `h4` for example one should be able to provide an icon for that.